### ISSUE TITLE
[Bug]: Creating server

### ISSUE THREAD

**Issue by ItzJustSamu** [FyreHost/FyreBot-issues#3]
### What happened?

two different commands both failed 

First for No value Present = 

/createserver name: test owner: ItzJustSamuel@gmail.com cpu: 0 ram: 1000 disk: 10000 ip: 192.168.1.3:50006 nest: Minecraft egg: Paper

Second command: Index: 0, Size: 0
/createserver name: test owner: ItzJustSamuel@gmail.com cpu: 0 ram: 1000 disk: 10000 ip: 192.168.1.3:50006 nest: 1 egg: 3

### Version

1.0.2 (Default)

### What setup are you seeing the bug on?

Pterodactyl (FyreBot Egg)

### Relevant log output

```shell
java.util.NoSuchElementException: No value present
        at java.util.Optional.get(Optional.java:135)
        at net.fyrehost.fyrebot.commands.CreateServerCommand.onCommand(CreateServerCommand.java:51)
        at net.fyrehost.fyrebot.Core.lambda$main$0(Core.java:97)
        at net.fyrehost.fyrebot.Core$$Lambda$721/0x0000000068543390.apply(Unknown Source)
        at discord4j.core.event.EventDispatcher.lambda$on$1(EventDispatcher.java:125)
        at discord4j.core.event.EventDispatcher$$Lambda$746/0x00000000e4078400.get(Unknown Source)
        at reactor.core.publisher.FluxDefer.subscribe(FluxDefer.java:46)
        at reactor.core.publisher.Flux.subscribe(Flux.java:8469)
        at reactor.core.publisher.FluxFlatMap$FlatMapMain.onNext(FluxFlatMap.java:426)
        at reactor.core.publisher.FluxDoFinally$DoFinallySubscriber.onNext(FluxDoFinally.java:130)
        at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.onNext(FluxPeekFuseable.java:210)
        at reactor.core.publisher.FluxHandleFuseable$HandleFuseableSubscriber.tryOnNext(FluxHandleFuseable.java:125)
        at reactor.core.publisher.FluxMapFuseable$MapFuseableConditionalSubscriber.tryOnNext(FluxMapFuseable.java:311)
        at reactor.core.publisher.FluxFilterFuseable$FilterFuseableConditionalSubscriber.tryOnNext(FluxFilterFuseable.java:367)
        at reactor.core.publisher.FluxPublishOn$PublishOnConditionalSubscriber.runAsync(FluxPublishOn.java:992)
        at reactor.core.publisher.FluxPublishOn$PublishOnConditionalSubscriber.run(FluxPublishOn.java:1067)
        at reactor.scheduler.forkjoin.ForkJoinPoolScheduler$DisposableWorkerTask.run(ForkJoinPoolScheduler.java:443)
        at reactor.scheduler.forkjoin.ForkJoinPoolScheduler$Worker.processTaskQueue(ForkJoinPoolScheduler.java:407)
        at reactor.scheduler.forkjoin.ForkJoinPoolScheduler$Worker$$Lambda$727/0x0000000068526990.run(Unknown Source)
        at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
        at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)




java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
        at java.util.ArrayList.rangeCheck(ArrayList.java:659)
        at java.util.ArrayList.get(ArrayList.java:435)
        at java.util.Collections$UnmodifiableList.get(Collections.java:1311)
        at net.fyrehost.fyrebot.commands.CreateServerCommand.onCommand(CreateServerCommand.java:49)
        at net.fyrehost.fyrebot.Core.lambda$main$0(Core.java:97)
        at net.fyrehost.fyrebot.Core$$Lambda$721/0x0000000068543390.apply(Unknown Source)
        at discord4j.core.event.EventDispatcher.lambda$on$1(EventDispatcher.java:125)
        at discord4j.core.event.EventDispatcher$$Lambda$746/0x00000000e4078400.get(Unknown Source)
        at reactor.core.publisher.FluxDefer.subscribe(FluxDefer.java:46)
        at reactor.core.publisher.Flux.subscribe(Flux.java:8469)
        at reactor.core.publisher.FluxFlatMap$FlatMapMain.onNext(FluxFlatMap.java:426)
        at reactor.core.publisher.FluxDoFinally$DoFinallySubscriber.onNext(FluxDoFinally.java:130)
        at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.onNext(FluxPeekFuseable.java:210)
        at reactor.core.publisher.FluxHandleFuseable$HandleFuseableSubscriber.tryOnNext(FluxHandleFuseable.java:125)
        at reactor.core.publisher.FluxMapFuseable$MapFuseableConditionalSubscriber.tryOnNext(FluxMapFuseable.java:311)
        at reactor.core.publisher.FluxFilterFuseable$FilterFuseableConditionalSubscriber.tryOnNext(FluxFilterFuseable.java:367)
        at reactor.core.publisher.FluxPublishOn$PublishOnConditionalSubscriber.runAsync(FluxPublishOn.java:992)
        at reactor.core.publisher.FluxPublishOn$PublishOnConditionalSubscriber.run(FluxPublishOn.java:1067)
        at reactor.scheduler.forkjoin.ForkJoinPoolScheduler$DisposableWorkerTask.run(ForkJoinPoolScheduler.java:443)
        at reactor.scheduler.forkjoin.ForkJoinPoolScheduler$Worker.processTaskQueue(ForkJoinPoolScheduler.java:407)
        at reactor.scheduler.forkjoin.ForkJoinPoolScheduler$Worker$$Lambda$727/0x0000000068526990.run(Unknown Source)
        at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
        at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ItzJustSamu** [comment]
Tried it with the UUID of the egg and nest and same thing

 java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
        at java.util.ArrayList.rangeCheck(ArrayList.java:659)
        at java.util.ArrayList.get(ArrayList.java:435)
        at java.util.Collections$UnmodifiableList.get(Collections.java:1311)
        at net.fyrehost.fyrebot.commands.CreateServerCommand.onCommand(CreateServerCommand.java:49)
        at net.fyrehost.fyrebot.Core.lambda$main$0(Core.java:97)
        at net.fyrehost.fyrebot.Core$$Lambda$721/0x0000000068543390.apply(Unknown Source)
        at discord4j.core.event.EventDispatcher.lambda$on$1(EventDispatcher.java:125)
        at discord4j.core.event.EventDispatcher$$Lambda$746/0x00000000e4078400.get(Unknown Source)
        at reactor.core.publisher.FluxDefer.subscribe(FluxDefer.java:46)
        at reactor.core.publisher.Flux.subscribe(Flux.java:8469)
        at reactor.core.publisher.FluxFlatMap$FlatMapMain.onNext(FluxFlatMap.java:426)
        at reactor.core.publisher.FluxDoFinally$DoFinallySubscriber.onNext(FluxDoFinally.java:130)
        at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.onNext(FluxPeekFuseable.java:210)
        at reactor.core.publisher.FluxHandleFuseable$HandleFuseableSubscriber.tryOnNext(FluxHandleFuseable.java:125)
        at reactor.core.publisher.FluxMapFuseable$MapFuseableConditionalSubscriber.tryOnNext(FluxMapFuseable.java:311)
        at reactor.core.publisher.FluxFilterFuseable$FilterFuseableConditionalSubscriber.tryOnNext(FluxFilterFuseable.java:367)
        at reactor.core.publisher.FluxPublishOn$PublishOnConditionalSubscriber.runAsync(FluxPublishOn.java:992)
        at reactor.core.publisher.FluxPublishOn$PublishOnConditionalSubscriber.run(FluxPublishOn.java:1067)
        at reactor.scheduler.forkjoin.ForkJoinPoolScheduler$DisposableWorkerTask.run(ForkJoinPoolScheduler.java:443)
        at reactor.scheduler.forkjoin.ForkJoinPoolScheduler$Worker.processTaskQueue(ForkJoinPoolScheduler.java:407)
        at reactor.scheduler.forkjoin.ForkJoinPoolScheduler$Worker$$Lambda$727/0x0000000068526990.run(Unknown Source)
        at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
        at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)

**Comment by mrxbox98** [comment]
Fixed in 3.0.7 hotfix
